Update persist documentation (#8837)
ballerina-platform/ballerina-dev-website: swan-lake/development-tutorials/ballerina-persist/persist-cli-tool.md

- The model definition file should contain at least one entity
- If you invoke the command twice, it will not fail. It will regenerate the files again.

## Generate the model definition by introspecting an existing database [Experimental]

>**Info:** The support for introspection is currently an experimental feature, and its behavior may be subject to change in future releases. Also, the support for introspection is currently limited to the MySQL data store.

> The command below generates the model definition for an existing database. This command is executed in the project root directory. The generated model definition can be used to generate the client API for the existing database after verifying it.

Reviewer: ```suggestion
The command below generates the data model for an existing database. The generated model definition can be used to generate the client API for the existing database after verifying it.
```

```

| Command Parameter |                                   Description                                    | Mandatory | Default Value |
|:-----------------:|:--------------------------------------------------------------------------------:|:---------:|:-------------:|
|    --datastore    | used to indicate the preferred data store. Currently, only 'mysql' is supported. |    No     |     mysql     |
|      --host       |                        used to indicate the database host                        |    Yes    |     None      |
|      --port       |                        used to indicate the database port                        |    No     |     3306      |
|      --user       |                        used to indicate the database user                        |    Yes    |     None      |
|    --database     |                        used to indicate the database name                        |    Yes    |     None      |

The file structure of the project after executing the command will be as follows.

```
rainier
   ├── persist
         └── model.bal   
   ├── Ballerina.toml
   ├── Config.toml
   └── main.bal
```

> This command will introspect the schema of the database and create a `model.bal` file with the entities and relations based on the schema of the database. The database configuration should be provided as command-line arguments.

Reviewer: ```suggestion
This command introspects the schema of the database and creates a `model.bal` file with the entities and relations based on the schema of the database. The database configuration should be provided as command-line arguments.
```

## Generate the data model by introspecting an existing database [Experimental]

>**Info:** The support for introspection is currently an experimental feature, and its behavior may be subject to change in future releases. Also, the support for introspection is currently limited to the MySQL data store.

The command below generates the data model for an existing database. The generated data model can be used to generate the client API for the existing database after verifying it.

```
$ bal persist pull --datastore mysql --host localhost --port 3306 --user root --database db
```

| Command Parameter |                                   Description                                    | Mandatory | Default Value |
|:-----------------:|:--------------------------------------------------------------------------------:|:---------:|:-------------:|
|    --datastore    | used to indicate the preferred data store. Currently, only 'mysql' is supported. |    No     |     mysql     |
|      --host       |                        used to indicate the database host                        |    Yes    |     None      |

> |      --port       |                        used to indicate the database port                        |    No     |     3306      |

Reviewer: shall we mandate the port here? as other databases have different port numbers

Author: As per the physical discussion, it was decided to keep the port non-mandatory since we can set the relevant default port for the particular data store if the port is not set, while we check the data store when loading the drivers. Closing this as resolved.
